Validation.py

```diff
@@ -38,6 +42,8 @@ def __del__(self):
         
     def plot2DVar(self, name, binxl, binxh, numbin, xIn, yIn, nbiny):
 
+        corr = cor.pearson_corr(xIn, yIn)
```

Comment:
Is this used anywhere? I assume you forgot to plot it on the histogram?

Reply:
If it isn't used lets remove it to save time

Reply:
That CC is indeed _not_ used anywhere, so I have removed it.